### DISCUSSION TITLE
Livewire only supports one HTML element per component. Multiple root elements detected for component: [filament.pages.auth.login]

### DISCUSSION THREAD

**Discussion by acubal** [filamentphp/filament#14218] [Help]
### Package

Panel builder

### Package Version

v3.2.111  

### How can we help you?

I already have existing news listing using livewire and it works fine. Then, I installed the filament & create a user according to the documentation [Filament Documentation](https://filamentphp.com/docs/3.x/panels/installation#installation). Then I went to the `/admin` and it triggered the error `Livewire only supports one HTML element per component. Multiple root elements detected for component: [filament.pages.auth.login]`. No changes were made to the blades. 

- Laravel v11.23.2 
- PHP 8.3.10 
- Livewire v3.5.6

**Comment by leandrocfe** [comment]
Does it happen in a Filament fresh install?

**Reply by acubal** [reply]
Yes sir. Despite removing the livewire components that I have before installing the filament.
